scripts/publish-npm.sh

```
@@ -0,0 +1,51 @@
+node "$ROOT/scripts/build-npm.mjs" --binaries "$BINARIES_DIR" --version "$VERSION" --out "$OUT"
+
+publish() {
+  dir="$1"
+  echo "Publishing $(basename "$dir")…"
+  # Provenance requires the OIDC token CI provides; it is silently unavailable
+  # (and must not be requested) on a developer machine.
+  prov=""
+  if [ -n "${CI:-}" ] && [ -n "${ACTIONS_ID_TOKEN_REQUEST_URL:-}" ]; then
+    prov="--provenance"
+  fi
+  # shellcheck disable=SC2086 -- $prov and $DRY_RUN are intentionally word-split
```

Shellcheck (0.11.0)

[error] Couldn't parse this shellcheck directive. Fix to allow more checks.

(SC1073)